### ISSUE TITLE
does pulsar source will ack the msg?

### ISSUE THREAD

**Issue by milkdrink** [risingwavelabs/risingwave#15492]
### Describe the bug

it seems the pulsar source consume message but doesn't ack the message ,so the pulsar backlog size will increase day by day.

![image](https://github.com/risingwavelabs/risingwave/assets/1709875/230d1d37-d691-4f01-b628-8af316fdd80e)
![image](https://github.com/risingwavelabs/risingwave/assets/1709875/ceb6f89a-a472-4734-898d-57d4665b3063)

**Comment by tabVersion** [comment]
Yes, RW does not commit back the offset back to pulsar broker. Can you share more about your use case and can message be managed by retention time, such as 7 days in kafka?

**Comment by NiuBlibing** [comment]
> Yes, RW does not commit back the offset back to pulsar broker. Can you share more about your use case and can message be managed by retention time, such as 7 days in kafka?

Same issue, there are some trouble when rw does not send ack. 

1. Monitor system is broken: pulsar-manager and dashboard will show all messages is backlog. 
2. We have to set a larger retention policy to avoid message cleaned before consumed due to  we don't know whether the messages is consumed correctly, It bring overload for disk space.

**Comment by tabVersion** [comment]
https://github.com/risingwavelabs/risingwave/pull/16733 this solution can also apply to pulsar 🤔

**Comment by NiuBlibing** [comment]
> #16733 this solution can also apply to pulsar 🤔

Using[ `cumulative_ack`](https://github.com/streamnative/pulsar-rs/blob/918c25a82b77bf55a1afde0e4a991faeb8757ecc/src/consumer/multi.rs#L225) ?
https://pulsar.apache.org/docs/3.2.x/concepts-messaging/#acknowledgment
